model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -585,7 +585,6 @@ object WorkspaceSettingTypes {
   sealed trait WorkspaceSettingType extends RawlsEnumeration[WorkspaceSettingType] {
     override def toString: String = getClass.getSimpleName.stripSuffix("$")
     override def withName(name: String): WorkspaceSettingType = WorkspaceSettingTypes.withName(name)
-    def defaultConfig(): WorkspaceSettingConfig
```

Comment:
Marc mentioned that this was a holdover from a previous refactor of the workspace settings, and since it's only used in tests now I took it out. Having it there added some confusion about what it was being used for/where this default config comes into play.